package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@babel/runtime": "7.10.1",
     "bluebird": "2.9.25",
-    "forest-express": "9.2.0",
+    "forest-express": "9.2.4",
```

Comment:
This is one of the bad versions of forest-express.
We need to wait and change this